### PR TITLE
py-plum-dispatch: add new package

### PR DESCRIPTION
**Pull request by ChristopherChristofi**

### CODE REVIEW

**Comment by ChristopherChristofi** [comment]
@spackbot fix style

**Comment by ChristopherChristofi** [comment]
@spackbot run pipeline

**Review by tldahlgren — COMMENTED** [review]
FYI. Confirmed the homepage and version sha256.  Adding @adamjstewart

**Review by adamjstewart — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-plum-dispatch/package.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2012-2023 Lawrence Livermore National Security, LLC and other
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-plum-dispatch/package.py]:
```suggestion
# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```
I think this is why the style tests are failing

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**